class.jetpack.php

```diff
@@ -7312,7 +7312,7 @@ public static function load_jetpack_gutenberg() {
 		if ( ! Jetpack::is_active() ) {
 			return;
 		}
-		if ( ! Jetpack::is_gutenberg_available() || ! apply_filters( 'jetpack_gutenberg', false ) ) {
+		if ( ! Jetpack::is_gutenberg_available() || ! apply_filters( 'jetpack_gutenberg', true ) ) {
```

Comment:
Looks like we missed it earlier, but should add a docblock for this filter.

Reply:
Good spot, both of you. Positioned and updated in 37b329f00dda4f255357944e2cb86d853b2960aa.